### ISSUE TITLE
Updates broken

### ISSUE THREAD

**Issue by cooler-king** [dart-lang/angular_analyzer_plugin#538]
Window x64
analyzer plugin 0.0.14
Dart 2.0.0-dev.43

With the filter set to whole project, errors, warnings and hints are not showing up as expected.  If I go to a specific HTML file, then sometimes the analyzer lines will come in a few seconds later, but sometimes not.  If I make an intentional error then the results for the edited file show up almost immediately.  If I back out of the intentional error, analyzer lines remain that were not there before (that is, now I can see the problems).  Results from other files disappear.

So basically to be certain to see analysis results right now I have to go to every individual HTML file, make an intentional error and then back out of it.  I can never see all of the errors from all of the files at the same time.

**Comment by MichaelRFairhurst** [comment]
> So basically to be certain to see analysis results right now I have to go to every individual HTML file, make an intentional error and then back out of it. I can never see all of the errors from all of the files at the same time.

Well that makes it pretty much unusable doesn't it!

This is probably a race condition where files are getting updated multiple times, and incorrectly at one point. Then there's some kind of ordering related to the file you're editing which lets it "win" over the other empty error sets. That sounds fancy, but I'm mostly just really confused at what could cause this.

Two questions you are free to decline:

Would you be able to give me some basic info like how many components there are in your project, and maybe one example of a component/template that has this problem?

Would you be able to do a video call to show me what it looks like? Or record a video? Feel free to reply that that question via email mfairhurst@google.com

Thanks for filing. I hadn't heard of anything going so drastically wrong, this is a big deal!

**Comment by cooler-king** [comment]
@MichaelRFairhurst example sequence sent to email.

**Comment by cooler-king** [comment]
After upgrading to dev.69 here is what I'm seeing.
I can get the analyzer to start according to the Analysis Server Diagnostics page.
No lints are reported by the plugin even if there are obvious errors in the HTML.
The analyzer plugin then 'disappears'... meaning that refreshing the Diagnostics page eventually shows no plugin active anymore, usually after a few seconds.  No exceptions are reported.

Summary:  plugin appears to start normally but displays no lints and stops after a short period without complaint.

**Comment by MichaelRFairhurst** [comment]
I think the analyzer is finally fully working + stable with Dart 2.

Please continue to file issues for anything that comes up while using it!

One comment, the auto-update mechanism is not working. You will likely have to remove `~/.dartServer/.plugin_manager`: https://github.com/MichaelRFairhurst/angular2-dart-analyzer#troubleshooting to get it to work. I'm looking into that now.
